src/client/client.ts

```diff
@@ -275,6 +281,65 @@ export class Client {
     return res;
   }
 
+  /**
+   * Batch publish messages to QStash.
+   */
+  public async batch(
+    req: PublishRequest[]
+  ): Promise<(PublishToUrlResponse | PublishToTopicResponse)[]> {
+    const messages = [];
+    for (const message of req) {
+      const headers = this.processHeaders(message);
+      const headerEntries: Record<string, string> = {};
+      for (const [key, value] of headers.entries()) {
+        headerEntries[key] = value;
+      }
+
+      messages.push({
+        destination: message.url ?? message.topic,
+        headers: headerEntries,
+        body: message.body,
+      });
+    }
+
+    const res = await this.http.request<
+      (PublishToUrlResponse | PublishToTopicResponse)[]
+    >({
+      path: ["v2", "batch"],
+      body: JSON.stringify(messages),
+      headers: {
+        "Content-Type": "application/json",
+      },
+      method: "POST",
+    });
+
+    return res;
+  }
+
+  /**
+   * Batch publish messages to QStash, serializing each body to JSON.
+   */
+  public async batchJSON<
+    TBody = unknown,
```

Comment:
If we do this `TBody extends BodyInit = BodyInit` we don't have to `@ts-ignore` at the end.

Reply:
Yessir